modules/ROOT/pages/manage-apis-with-service-mesh.adoc

```diff
@@ -31,6 +32,9 @@ image::apply-client-enforcement-policy-service-mesh.png[]
 
 For more information about how to apply policies, see the specific policy documentation.
 
+== Viewing Analytics Data for Your APIs
+
+You can monitor the usage of your APIs to determine various aspects of their performance. For more information about how to view the analytics data, see xref:monitoring::api-analytics-dashboard.adoc[API Analytics Dashboard].
```

Comment:
Please link to this page - https://docs.mulesoft.com/api-manager/2.x/analytics-landing-page

Reply:
Done! Thanks!